### ISSUE TITLE
Panics on Master?

### ISSUE THREAD

**Issue by catsby** [heroku/busl#24]
I'm checking out some of the logging updates and I'm getting a panic locally. 

Run this in a terminal:

```
$ REDIS_URL=<redis url> PORT=5000 godep go run main.go
```
(or however you run busl locally)

Then curl to create a stream:

```
$ curl -X POST localhost:5001/streams
b5637d1c40a8003c8084c6eb7147ceb0
```

Then curl to subscribe:

```
$ curl localhost:5001/streams/b5637d1c40a8003c8084c6eb7147ceb0 
curl: (52) Empty reply from server
```

In the terminal running busl, I get this:

```
busl source=development pid=6684 connecting to redis: {redis  <nil> 127.0.0.1:6379   }
Starting busl...
busl source=development pid=6684 signals.await signals=[urgent I/O condition]
busl source=development pid=6684 http.graceful.await
busl source=development pid=6684 count#http.status.4xx=1 status=404
busl source=development pid=6684 count#mkstream.create.success=1
busl source=development pid=6684 count#http.status.2xx=1 status=200
busl source=development pid=6684 http: panic serving [::1]:52063: interface conversion: *util.responseLogger is not http.CloseNotifier: missing method CloseNotify
goroutine 13 [running]:
net/http.func·011()
	/Users/clint/Developer/Cellar/go/1.4.1/libexec/src/net/http/server.go:1130 +0xbb
github.com/heroku/busl/server.sub(0x59d7e8, 0xc20801e9c0, 0xc208030b60)
	/Users/clint/Projects/Go/src/github.com/heroku/busl/server/server.go:98 +0x9c
github.com/heroku/busl/server.func·003(0x59d7e8, 0xc20801e9c0, 0xc208030b60)
```

**Comment by cyx** [comment]
Good catch, I'll probably write a test to catch regressions.

**Comment by catsby** [comment]
Fixed with #25
